__tests__/endpoint.test.js

```diff
@@ -82,3 +82,40 @@ describe("/api/articles/", () => {
       });
   });
 });
+describe("/api/articles", () => {
+  test(`when all articles is requested, reponds with all articles in ascending order by creating date and no body`, () => {
+    return request(app)
+      .get("/api/articles")
```

Comment:
we would also like a length check to make sure our toBeSortedBy is checking a populated array as an empty array or an array with 1 thing in would always be sorted by default